.github/workflows/ci.yml

```diff
@@ -49,11 +49,16 @@ jobs:
             sudo apt-get update
             sudo apt-get -y install libvips-tools ffmpeg mediainfo tesseract-ocr qpdf imagemagick exiftool
 
+        # Note: as of early 2024, 'yarn install' causes GitHub to install node 18.19.1.
+        # Whatever version is installed, make sure it is permitted in file
+        # package.json under key "engines".
```

Comment:
Sorry, a nit-pick, but it's not that "yarn install" causes github to install node, I don't think, but that Github Actions VM's have a pre-installed node, that `yarn install` _uses_, that has to match requirements.